lib/escape.js

```diff
@@ -1,21 +1,12 @@
-const normalizeArgs = (file, args = []) => {
-	if (!Array.isArray(args)) {
-		return [file];
-	}
-
-	return [file, ...args];
+export const joinCommand = (filePath, rawArgs) => {
+	const fileAndArgs = [filePath, ...rawArgs];
+	const command = fileAndArgs.join(' ');
+	const escapedCommand = fileAndArgs.map(arg => escapeArg(arg)).join(' ');
+	return {command, escapedCommand};
 };
 
-const NO_ESCAPE_REGEXP = /^[\w.-]+$/;
-
-const escapeArg = arg => {
-	if (typeof arg !== 'string' || NO_ESCAPE_REGEXP.test(arg)) {
-		return arg;
-	}
+const escapeArg = arg => typeof arg === 'string' && !NO_ESCAPE_REGEXP.test(arg)
+	? `"${arg.replaceAll('"', '\\"')}"`
+	: arg;
 
-	return `"${arg.replaceAll('"', '\\"')}"`;
-};
-
-export const joinCommand = (file, rawArgs) => normalizeArgs(file, rawArgs).join(' ');
-
-export const getEscapedCommand = (file, rawArgs) => normalizeArgs(file, rawArgs).map(arg => escapeArg(arg)).join(' ');
+const NO_ESCAPE_REGEXP = /^[\w.-]+$/;
```

Comment:
This refactoring (and the one in `index.js`) is not strictly related to this PR. I can split it into another PR if you want, or leave it in this one if this is ok with you.